### DISCUSSION TITLE
Conflict with `dotenv-safe`?

### DISCUSSION THREAD

**Discussion by dalisoft** [uNetworking/uWebSockets.js#168] [Q&A] [answered]
Hi @alexhultman 

I catched one issue unexpectedly.

This library conflicts with `dotenv-safe`.

```js
// import envSafe from 'dotenv-safe';
import uWS from 'uWebSockets.js';

const app = uWS.App();
// envSafe.config();

app.listen(3000);
```

This code runs, when remove comments part and use `dotenv-safe`, this project fails to listen and can't make it work anyway. Is there any way force it work?

**Comment by ghost** [comment]
You are not following documented API

**Comment by dalisoft** [comment]
This is snippet only, all of your examples doesn't work when using `dotenv`, `dotenv-safe` =)

**Comment by subversivo58** [comment, marked as answer]
Hi @dalisoft 

`dotenv-safe` uses `dotenv` than all values return as `{String}` type

This library requires that the **"port"** parameter be specified as `{Number}` type

View rows [129,131](https://github.com/uNetworking/uWebSockets.js/blob/master/docs/index.d.ts#L129,L131)

You can use `{Number}` and for your variables ... it's what I do:


```javascript
require('dotenv').config()

const uWS = require('uWebSockets.js')

// Listen server requiries `{Number}` type for "port" parameter of server
const PORT = Number(process.env.PORT) || 3000

const app = uWS.App()

// HTTP routes
app.any('/*', (res, req) => {})

// Server listen
app.listen(PORT, token => {
    if ( token ) {
        console.log('Runing on port: ' + PORT)
    } else {
        console.log('Failed to listen to port ' + PORT)
    }
})
```

**Comment by ghost** [comment]
Thanks @subversivo58
